### PR TITLE
add OhMyCrypt

### PR DESCRIPTION
**Pull request by OhMyCrypt**
GM, this is an addition to the PR posted on the DefiLlama-adapters repo. 

Our project is launching in a few hours on Base (explanation to the low volume)

Thanks

### CODE REVIEW

**Comment by llamatester** [comment]
The OhMyCrypt adapter exports: 
        
 
 _______________________________________
    ------ ERROR ------
    dexs/OhMyCrypt/index.ts(13,23): error TS7006: Parameter 'weiAmount' implicitly has an 'any' type.
    
    Done in 5.35s.

**Comment by llamatester** [comment]
The OhMyCrypt adapter exports: 
        
 
 _______________________________________
    Dexs for 25/9/2023
    _______________________________________
    
    
    
    Done in 3.39s.

**Comment by dtmkeng** [comment]
set start time `1695513600`

File: dexs/OhMyCrypt/index.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+// Fetch function to query the subgraphs
+const graphs = (graphUrls: ChainEndpoints) => {
+    return (chain: Chain) => {
+      return async (timestamp: number) => {
+        const todaysTimestamp = getTimestampAtStartOfDayUTC(timestamp);
+        const graphQueryusdcday = gql`
+          query {
+            activeOrders(where: {timestamp_gte: ${todaysTimestamp}, t2:"0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913" }) {
```
Comment (dtmkeng) [inline dexs/OhMyCrypt/index.ts]: could you change query like this `https://github.com/DefiLlama/dimension-adapters/blob/master/fees/curve.ts#L32`
fiter yesterday data

File: dexs/OhMyCrypt/index.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          const graphQueryusdcalltime = gql`
+          query {
+            activeOrders(where: {t2:"0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913"}) {
+              t2a
+            }
+          }
+          `;
+           const graphQuerydaiday = gql`
+           query {
+             activeOrders(where: {timestamp_gte: ${todaysTimestamp}, t2:"0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb" }) {
```
Comment (dtmkeng) [inline dexs/OhMyCrypt/index.ts]: could you change query like this https://github.com/DefiLlama/dimension-adapters/blob/master/fees/curve.ts#L32
fiter yesterday data

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The OhMyCrypt adapter exports: 
        
 
 _______________________________________
    Dexs for 25/9/2023
    _______________________________________
    
    
    
    Done in 3.37s.

**Comment by llamatester** [comment]
The OhMyCrypt adapter exports: 
        
 
 _______________________________________
    Dexs for 25/9/2023
    _______________________________________
    
    
    
    Done in 3.42s.

**Pull request closed without merging**